terminal/bundles/org.eclipse.terminal.view.ui/src/org/eclipse/terminal/view/ui/internal/TerminalService.java

```diff
@@ -268,11 +266,12 @@ public IStatus run(TerminalViewId tvid, String title, ITerminalConnector connect
 						ILog.get().log(e.getStatus());
```

Comment:
Should this exception really be suppressed here or should we retrow it?

If not I would suggest adding a message to the log e.g.

ILog.get().error("Can't restore view", e);

Reply:
I added another new commit that logs all errors instead of silently discarding them.